utils_cv/detection/plot.py

```diff
@@ -15,7 +15,7 @@
 import matplotlib.pyplot as plt
 
 from .references.coco_eval import CocoEvaluator
-from .bbox import AnnotationBbox
+from .bbox import _Bbox, AnnotationBbox, DetectionBbox
```

Comment:
[minor] Should Bbox be private or not? Ie. since it is used here in this fct, do we think the user will also need it?